server/src/patches/withFilter.ts

```diff
@@ -1,27 +0,0 @@
-import { GraphQLResolveInfo } from 'graphql';
-import { withFilter as withFilterOriginal } from 'graphql-subscriptions';
-
-type FilterFn<TParent = {}, TContext = {}, TArgs = {}> = (
-  parent: TParent,
-  args: TArgs,
-  context: TContext,
-  info: GraphQLResolveInfo
-) => boolean | Promise<boolean>;
-
-type ResolverFn<TResult, TParent, TContext, TArgs> = (
-  parent: TParent,
-  args: TArgs,
-  context: TContext,
-  info: GraphQLResolveInfo
-) => Promise<TResult> | TResult;
-
-// https://github.com/apollographql/graphql-subscriptions/issues/261#issuecomment-1246300357
-export const withFilter = withFilterOriginal as unknown as <
```

Comment:
Were you able to get subscription types working without this type override?

Reply:
yup!